controllers/bsl.go

```diff
@@ -60,6 +60,11 @@ func (r *DPAReconciler) ValidateBackupStorageLocations(log logr.Logger) (bool, e
 			default:
 				return false, fmt.Errorf("invalid provider")
 			}
+
+			_, err := r.resticCustomCAEnabled(*bslSpec.Velero, &dpa)
```

Comment:
This function is supposed to be a validate function, right ? The boolean returned by this function should not be ignored, or else how would the reconcile loop break if validation fails ?

Reply:
The function validates the custom CA field, where the only invalid condition is specifying "insecureSkipTLSVerify=false" without including a caCert. The returned boolean indicates whether or not "insecureSkipTLSVerify" is set, which is used later to decide if the custom CA field should be passed to restic. In this context, it does not matter if it is set to true or false, just that the configuration is valid (indicated by a nil err).

I guess it could be split up into separate validation and isEnabled functions?

Reply:
Reading it again, I agree with @mrnold . Validate does not need to know the bool returned by this function, only if it errors.